angr/analyses/propagator/propagator.py

```diff
@@ -156,6 +164,8 @@ def copy(self):
             self.arch,
             replacements=self._replacements.copy(),
             prop_count=self._prop_count.copy(),
+            only_consts=self._only_consts
+
```

Comment:
Why this extra blank line?

Also, please end the previous line (167) with a comma at the end. That will make future git diffs look better.